(try 
    (throw e1 "msg1") 
    (catch e2 (print e2))
)
;err:
error: e1, msg1
at TRY: (throw e1 "msg1")
at main: (try (throw e1 "msg1") (catch e2 (print e2)))
;eof

(try 
    (try 
        (throw e1 "msg1") 
        (catch e2 (print e2))
    ) 
    (catch ?e (print ?e))
)
;=>nil
;out:
error: e1, msg1
;eof

